src/sample/Main.java

```diff
@@ -7,31 +7,21 @@
 import javafx.stage.Stage;
 
 import java.io.IOException;
-import java.util.logging.Level;
-import java.util.logging.Logger;
 
+// changing stages (Main).
 public class Main extends Application {
 
-    private static Stage stage;
-
     @Override
-    public void start(Stage primaryStage) throws Exception{
-        Main.stage = primaryStage;
-        changeView("View/firstScreen.fxml");
-        primaryStage.setTitle("Welcome to the Sleepyhead App");
-        primaryStage.setResizable(false);
```

Comment:
I think this line is necessary

Reply:
21 is just for the title of the page. 22 is for that user can't change the size of the window. It is optional too

Reply:
I know what are 21 and 22 lines do. 
I haven't made the first screen resizable so line 21 is necessary.  If you made the window adjustable it is not necessary.

Reply:
Sorry, my bad. I mean 22 is necessary. If you resize the size of first screen, you will understand what I mean. Other things are fine for me. plz merge it :)